REQUIRE

```diff
@@ -1,2 +1,2 @@
 julia 0.3
-Compat 0.7.3
+Compat 0.7.9
```

Comment:
this is not the right version. 0.7.15 introduced `String` in Compat, which you need to import. https://github.com/JuliaLang/Compat.jl/commit/4c8c62d275fd11273124fcf4d76f2513eafa0ccc